sdk/containerregistry/azure-containerregistry/README.md

```diff
@@ -56,12 +56,89 @@ For more information please see [Container Registry Concepts](https://docs.micro
 
 ## Examples
 
-<!-- Pending Sample Creation -->
+The following sections provide several code snippets covering some of the most common ACR Service tasks, including:
+
+- [List repositories](#list-repositories)
+- [List tags with anonymous access](#list-tags-with-anonymous-access)
+- [Set artifact properties](#set-artifact-properties)
+- [Delete images](#delete-images)
+
+Please note that each sample assumes there is a `CONTAINERREGISTRY_ENDPOINT` environment variable set to a string containing the `https://` prefix and the name of the login server, for example "https://myregistry.azurecr.io".
+
+### List repositories
+
+Iterate through the collection of repositories in the registry.
+
+```python
+end_point = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+client = ContainerRegistryClient(end_point, DefaultAzureCredential(), audience="https://management.azure.com")
+
+with client:
+    # Iterate through all the repositories
+    for repository_name in client.list_repository_names():
+        print(repository_name)
+
+client.close()
+```
+
+### List tags with anonymous access
+
+Iterate through the collection of tags in the repository with anonymous access.
+
+```python
+end_point = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+client = ContainerRegistryClient(end_point, DefaultAzureCredential(), audience="https://management.azure.com")
+
+manifest = client.get_manifest_properties("library/hello-world", "latest")
+print(manifest.repository_name + ": ")
+for tag in manifest.tags:
+    print(tag + "\n")
+
+client.close()
+```
+
+### Set artifact properties
+
+Set properties of an artifact.
+
+```python
+end_point = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+client = ContainerRegistryClient(end_point, DefaultAzureCredential(), audience="https://management.azure.com")
+
+# Set permissions on the v1 image's "latest" tag
+client.update_manifest_properties(
+    "library/hello-world",
+    "latest",
+    can_write=False,
+    can_delete=False
+)
+
+client.close()
+```
+
+### Delete images
+
+Delete images older than the first three in the repository.
+
+```python
+end_point = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+client = ContainerRegistryClient(end_point, DefaultAzureCredential(), audience="https://management.azure.com")
+
+for repository in client.list_repository_names():
+    manifest_count = 0
+    for manifest in client.list_manifest_properties(repository, order_by=ManifestOrder.LAST_UPDATE_TIME_DESCENDING):
+        manifest_count += 1
+        if manifest_count > 3:
+            print("Deleting {}:{}".format(repository, manifest.digest))
+            client.delete_manifest(repository, manifest.digest)
+
+client.close()
+```
 
 ## Troubleshooting
 
 ### General
-Form Recognizer client library will raise exceptions defined in [Azure Core][azure_core_exceptions].
+Form ACR client library will raise exceptions defined in [Azure Core][azure_core_exceptions].
```

Comment:
Should we spell the full-service name instead of just the acronym?
```suggestion
ACR client library will raise exceptions defined in [Azure Core][azure_core_exceptions].
```